fe/fe-core/src/main/java/com/starrocks/common/ErrorCode.java

```diff
@@ -122,6 +122,8 @@ public enum ErrorCode {
     ERR_EMPTY_PARTITION_IN_TABLE(1748, new byte[] {'H', 'Y', '0', '0', '0'},
             "data cannot be inserted into table with empty partition. " +
                     "Use `SHOW PARTITIONS FROM %s` to see the currently partitions of this table. "),
+    ERR_SECURE_TRANSPORT_REQUIRED(3159, new byte[] {'H', 'Y', '0', '0', '0'},
```

Comment:
The error code before 5000 is deprecated.
You can add this error code at
5200 - 5299: Authentication and Authorization
https://github.com/StarRocks/starrocks/blob/main/fe/fe-core/src/main/java/com/starrocks/common/ErrorCode.java#L246. 
And use 5205.

Reply:
@gengjun-git it's probably better to clarify the deprecation intention by updating this comment line in the file
```
 // Try our best to compatible with MySQL's
```

Reply:
fyi for @kevincai